PowerEditor/src/NppDarkMode.cpp

```diff
@@ -361,10 +362,26 @@ namespace NppDarkMode
 		_options = configuredOptions();
 
 		initExperimentalDarkMode();
+		initAdvancedOptions();
+
+		if (NppDarkMode::isWindowsModeEnabled())
```

Comment:
change to `if (NppDarkMode::isWindowsModeEnabled() && !NppDarkMode::isExperimentalSupported())` to be sure

Reply:
@ozone10 
Is it rather `if (NppDarkMode::isWindowsModeEnabled() && NppDarkMode::isExperimentalSupported())` ?

Reply:
@donho 
you're right, my mistake, I did not noticed that `!` 
we can also use `NppDarkMode::isWindows10()` instead of `NppDarkMode::isExperimentalSupported()`

Reply:
@donho 
this seems to be best way to handle it
```
void initDarkMode()
{
	_options = configuredOptions();

	initExperimentalDarkMode();
	initAdvancedOptions();

	g_isAtLeastWindows10 = NppDarkMode::isWindows10();

	if (!g_isAtLeastWindows10)
	{
		NppDarkMode::setWindowsMode(false);
	}

	if (NppDarkMode::isWindowsModeEnabled())
	{
		NppParameters& nppParam = NppParameters::getInstance();
		NppGUI& nppGUI = nppParam.getNppGUI();
		nppGUI._darkmode._isEnabled = NppDarkMode::isDarkModeReg() && !IsHighContrast();
		_options.enable = nppGUI._darkmode._isEnabled;
		_options.enableMenubar = _options.enable;
	}

	setDarkMode(_options.enable, true);

	using PWINEGETVERSION = const CHAR* (__cdecl *)(void);

	PWINEGETVERSION pWGV = nullptr;
	auto hNtdllModule = GetModuleHandle(L"ntdll.dll");
	if (hNtdllModule)
	{
		pWGV = reinterpret_cast<PWINEGETVERSION>(GetProcAddress(hNtdllModule, "wine_get_version"));
	}

	g_isWine = pWGV != nullptr;
}
```